packages/gatsby-remark-prismjs/README.md

```diff
@@ -16,7 +16,16 @@ plugins: [
     resolve: `gatsby-transformer-remark`,
     options: {
       plugins: [
-        `gatsby-remark-prismjs`,
+        {
+          resolve: `gatsby-remark-prismjs`,
+          options: {
+            // Class prefix for <pre> tags containing syntax highlighting;
+            // Defaults to 'language-'.
+            // If you use Prism directly within your site,
```

Comment:
Perhaps rewrite this sentence slightly so makes clear this option is only needed when loading prismjs into the browser and this is not very common.